cmd/main.go

```diff
@@ -129,6 +147,10 @@ func main() {
 		FeatureOptions:   featureOptions,
 	}
 
+	if *runController && *enableMultishare && *enableStateful {
+		runMultishareReconciler(config)
```

Comment:
can we move this reconciler inside the [GCFSDriver.run()](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/pkg/csi_driver/gcfs_driver.go#L228) to keep the reconciler controllers in one place?

Reply:
in the current code I'm testing I did moved this func into the GCFSDriver. it's a bit hard to port it back to here though

Reply:
hard to port back here you mean in this PR?